### PR TITLE
Get api host and port from env variables

### PR DESCRIPTION
**Pull request by isabelcosta**
## Why

When running Docker container on my local machine I was getting the following error:
```
~ > curl -i http://localhost:8080/feed  
curl: (52) Empty reply from server
```

After some debugging, I noticed it was because the server is running on localhost within the container, apparently unreachable from the host machine. Apparently using 0.0.0.0 host will expose the API to my host. 👀 https://github.com/moby/moby/issues/2522

## What

Added ability to get Host and Port from the environment variables.
Added instructions on how to run docker on the README.
Added `.env` to `.gitignore`.

## Testing

Using the commands from the README to run the docker container + setting up the `.env` file I get a successful response on my host machine.
```
~ > curl -i http://localhost:8080/feed    
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Content-Type
Access-Control-Allow-Origin: *
Content-Type: application/json
Date: Mon, 21 Nov 2022 21:35:00 GMT
Content-Length: 2

{}
```

### CODE REVIEW

File: api/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.env
```
Comment (sansaid) [inline api/.gitignore:1]: How come we need to ignore this?

File: api/main.go [hunk 2]
```diff
@@ -61,6 +61,22 @@ func run(ctx context.Context, store handler.Store) error {
 	}
 }
 
+func getAddress() string {
+	var host, port string
+	var exists bool
+
+	host, exists = os.LookupEnv("HOST")
+	if !exists {
+		host = "localhost"
```
Comment (sansaid) [inline api/main.go:70]: I think the default should be `0.0.0.0`

**Comment by sansaid** [comment]
Hey @isabelcosta , thanks for the PR! I left some comments, but I'm going to merge your changes in so that I can get the app working. :)

File: api/main.go (hunk 2 above)
Comment (isabelcosta) [reply api/main.go:70]: Good point @sansaid ! Thank you for pointing this out :)

**Pull request merged**